vue/src/vueTextMask.js

```diff
@@ -1,24 +1,38 @@
 import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
 
 export default {
-  textMaskInputElement: null,
```

Comment:
@lozjackson I have been unable to replicate this error in Chrome on a Mac. Could you please provide more details